tests/cli/polyctl_test.py

```
@@ -0,0 +1,227 @@
+                                      {protocol.M_OVERLAY_SEND: {"queued": True}})
+        self.assertEqual(rc, 0)
+        self.assertEqual(dict(server.received)[protocol.M_OVERLAY_SEND],
+                         {"files": ["a.png", "b.png"]})
+
+
+class ImportGuardTest(unittest.TestCase):
+    def test_cli_imports_without_pyqt5(self):
+        import subprocess
+        code = ("import sys; sys.modules['PyQt5'] = None; "
+                "import polyhost.cli.polyctl; print('ok')")
+        proc = subprocess.run([sys.executable, "-c", code],
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)